src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -1994,28 +1994,6 @@ CommandLineConfig setExterns(List<String> externs) {
       return this;
     }
 
-    private final List<String> js = new ArrayList<>();
```

Comment:
Are you removing this stuff because it appears to be unused? I think it's used from the internal command line runner.